gradle.properties

```diff
@@ -15,6 +16,7 @@ springVersion=4.2.5.RELEASE
 httpClientVersion=4.5.1
 jedisVersion=2.7.3
 springDataMongoVersion=1.8.2.RELEASE
+springDataCouchbaseVersion=2.0.0.RELEASE
```

Comment:
this could be probably upgraded to `2.1.4` (if Spring Session is currently using Spring Data Hopper @rwinch ?)